contracts/src/StakeTableV2.sol

```diff
@@ -599,15 +680,23 @@ contract StakeTableV2 is StakeTable, PausableUpgradeable, AccessControlUpgradeab
 
     /// @notice Set the minimum interval between commission updates
     /// @param newInterval The new minimum interval in seconds
-    function setMinCommissionUpdateInterval(uint256 newInterval) external virtual onlyOwner {
+    function setMinCommissionUpdateInterval(uint256 newInterval)
+        external
+        virtual
+        onlyRole(DEFAULT_ADMIN_ROLE)
```

Comment:
For my understanding. Given the changes in this PR if we kept `onlyOwner` that should be equivalent right? Since we are enforcing a single admin in this contract should we keep calling that admin owner and use `onlyOwner` for clarity?